style/properties/shorthands/text.mako.rs

```diff
@@ -7,23 +7,21 @@
 <%helpers:shorthand name="text-decoration"
                     engines="gecko servo"
                     flags="SHORTHAND_IN_GETCS"
-                    sub_properties="text-decoration-line
-                    ${' text-decoration-style text-decoration-color text-decoration-thickness' if engine == 'gecko' else ''}"
+                    sub_properties=" text-decoration-color text-decoration-line text-decoration-style                     
```

Comment:
Still weird spacing:

```suggestion
                    sub_properties="text-decoration-color text-decoration-line text-decoration-style
```